waltz-client/src/main/java/com/wepay/waltz/client/internal/network/WaltzNetworkClient.java

```diff
@@ -203,6 +252,8 @@ public void onChannelActive() {
                 for (Partition partition : partitions.values()) {
                     networkClientCallbacks.onMountingPartition(WaltzNetworkClient.this, partition);
                 }
+
+                checkServerToStorageConnectivity();
```

Comment:
This doesn't work. What we need to do here is to resubmit `CheckStorageConnectivityRequest` if `checkConnectivityRef` has a future.
`checkServerToStorageConnectivity()` **won't** resubmit the request if there **is** a future.